tests/css/test_help_text.py

```diff
@@ -0,0 +1,123 @@
+import pytest
```

Comment:
These tests aren't as precise as they could be - they ensure the error help text code is exercised and the output looks "reasonable". I think they're at a decent balance between value and maintainability overhead.

Reply:
I concur. I imagine we will tweak the working of the errors fairly regularly.